data/sql_updates/candidate_history.sql

```diff
@@ -66,7 +66,8 @@ left join disclosure.cand_inactive inactive on
     fec_yr.cand_id = inactive.cand_id and
     fec_yr.fec_election_yr < inactive.election_yr
 left join staging.ref_pty ref_party on fec_yr.cand_pty_affiliation = ref_party.pty_cd
-where max_cycle >= :START_YEAR
+where max_cycle >= :START_YEAR and
+    fec_yr.cand_id not in (select distinct cmte_id from unverified_filers_vw where cmte_id similar to '(P|S|H)%')
```

Comment:
Ah, that makes sense actually.  Because I believe as written that where statement will always evaluate to false.  Because there is no chance to select a candidate_id that is equal to any committee id.  Yeah we can pair on it if you like.  But think you'll have to relate candiate_id's via `ofec_cand_cmte_link`.